client/python/tests/test_factory.py

```diff
@@ -96,10 +97,19 @@ def test_transport_decorator_registers(join, listdir, yaml):
     assert isinstance(transport, FakeTransport)
 
 
-@patch.dict(os.environ, {"OPENLINEAGE_DISABLED": "true"})
-def test_env_disables_client():
-    transport = get_default_factory().create()
-    assert isinstance(transport, NoopTransport)
+@pytest.mark.parametrize("env_var_value, should_be_noop", [
+    ("true", True),
+    ("True", True),
+    ("TRUE", True),
+    ("false", False),
+    ("False", False),
+    ("FALSE", False),
```

Comment:
If this is going to come from an environment variable, does that imply it's always a string? Or can these vars also come in as a True or False boolean?

Reply:
Sorry, meant to comment on that. Yes, [getenv always returns a string](https://docs.python.org/3/library/os.html#os.getenv).